packages/components/src/text-input/text-input.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * WordPress dependencies
+ */
+import { useMergeRefs } from '@wordpress/compose';
+
+/**
+ * Internal dependencies
+ */
+import { contextConnect } from '../ui/context';
+import { View } from '../view';
+import TextInputArrows from './text-input-arrows';
+import TextInputSteppers from './text-input-steppers';
+import { useTextInput } from './hooks/use-text-input';
+import * as styles from './styles';
+
+/**
+ *
+ * @param {Object} props
```

Comment:
This [was formally](https://github.com/ItsJonQ/g2/blob/main/packages/components/src/TextInput/TextInput.js#L13):

```js
* @param {import('@wp-g2/create-styles').ViewOwnProps<import('./types').Props, 'input'>} props
```

There was no `TextInput.d.ts` declaration file in https://github.com/ItsJonQ/g2/tree/main/packages/components/types/components so I'm a bit unsure about how to approach things. 

Plus my Typescript needs a little refresher :)